students/kevin_t/Lesson5/database.py

```diff
@@ -0,0 +1,110 @@
+""" This module imports .csv files and creates a relational database using MongoDB"""
+import os
+import csv
+from pymongo import MongoClient
+
+class MongoDBConnection():
+    """MongoDB Connection"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+        """ be sure to use the ip address not name for local windows"""
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
+
+def import_data(directory_name, product_file, customer_file, rental_file):
+    """Brings in csv files, counts customers, products and rentals as well as errors"""
+    customer_errors = 0
+    product_errors = 0
+    rental_errors = 0
+
+    customer_count = 0
+    product_count = 0
+    rental_count = 0
+
+    mongo = MongoDBConnection()
+
+    with mongo:
+        database = mongo.connection.media
+
+        customer = database["customer"]
+        product = database["product"]
+        rental = database["rental"]
+
+        customer.drop
+        product.drop
+        rental.drop
+
+        try:
+            with open(os.path.join(directory_name, customer_file)) as csvfile:
+                customer_reader = csv.reader(csvfile, delimiter=',')
+                for row in customer_reader:
+                    customer_count += 1
+                    customer_info = {"user_id": row[0], "name": row[1], "address": row[2],
+                                     "phone_number": row[3], "email": row[4]}
+                    customer.insert_one(customer_info)
+        except:
```

Comment:
Avoid using general exceptions, the risk being that new issues not seen during development could end up being masked within the _catch_all_ handling. Replace all general exceptions with specific ones or remove the _try/except_ block altogether.